charts/gmsa/values.yaml

```diff
@@ -51,3 +51,5 @@ securityContext: {}
 tolerations: []
 qps: 30.0
 burst: 50
+randomHostname: 
+  enabled: false
```

Comment:
Does ConfigMap support this structure? 'enableRandomHostname'?

Reply:
This is a helm values file so we can have any structure we want here.  We are missing the mapping this value to the deployment file here: https://github.com/kubernetes-sigs/windows-gmsa/blob/2a3fe46e69529d71850bb6fcb022c5dcd6905482/charts/gmsa/templates/deployment.yaml#L49-L59

Reply:
will update, wondering in the beginning as well